### DISCUSSION TITLE
news/release-8.4/

### DISCUSSION THREAD

**Discussion by giscus** [PrismLauncher/prismlauncher.org#530] [News]
# news/release-8.4/

Now with extra TNT!

https://prismlauncher.org/news/release-8.4/

**Comment by ZekeZDev** [comment]
wow such an amazing release :D

**Reply by Lgmrszd** [reply]
Yayyy cool release !!! (not me just testing out comment section)

**Reply by Samalando** [reply]
Love it!

**Reply by BeansBeefBroccoli** [reply]
I agree.

**Comment by BeansBeefBroccoli** [comment]
Hurrah! 🎉

**Comment by KTrain5169** [comment]
such an amazing comment section

**Comment by ajmod73** [comment]
:3

**Reply by balt-dev** [reply]
:3

**Comment by Sheltr64** [comment]
cool
